app/utils/Process.js

```diff
@@ -148,6 +149,19 @@ class Process extends EventEmitter {
 
     getSave = () => 100 - ((this.size / this.originalSize) * 100);
 
+    _copySourceImage = () => {
+        const fileExtension = path.extname(this.file.path);
+        const newPath = this.file.path.replace(fileExtension, ` (1)${fileExtension}`);
```

Comment:
Use [path.parse](https://nodejs.org/api/path.html#path_path_parse_path) to get the path properties and use create the `newPath`.

```js
const parhProps = path.parse(this.file.path);
const newName = `${parhProps.name}(1)${parhProps.ext}`;
const newPath = path.resolve(parhProps.dir, newName);
```